tests/template_reaper_test.go

```diff
@@ -0,0 +1,72 @@
+package tests
+
+import (
```

Comment:
Is this test the same as the one from #99? If so does it make sense to remove it from this PR?

Reply:
Same as above.